judge/views/submission.py

```diff
@@ -181,8 +181,10 @@ def get(self, request, *args, **kwargs):
 @require_POST
 def abort_submission(request, submission):
     submission = get_object_or_404(Submission, id=int(submission))
-    if (not request.user.is_authenticated or (submission.was_rejudged or (request.profile != submission.user)) and
-            not request.user.has_perm('judge.abort_any_submission')):
+    if not request.user.is_authenticated:
```

Comment:
Note: this `if` is actually unecessary (`not request.user.has_perm('judge.abort_any_submission')` will be `True, `and `request.profile != submission.user` will also be `True`). Should it be removed or should we keep it in for clarity?

Reply:
It used to be needed from before the `request.profile` days, just drop it.

Reply:
Done.